internal/rpcserver/rpcserver.go

```diff
@@ -6541,6 +6541,9 @@ type Config struct {
 
 	// FiltererV2 defines the V2 filterer for the RPC server to use.
 	FiltererV2 FiltererV2
+
+	// BanMgr defines the peer manager for the RPC server to use.
+	BanMgr BanManager
```

Comment:
Actually, I don't think the `rpcserver` needs this of the associated interface at all since it's not used.